integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -218,12 +228,12 @@ describe("App", () => {
   describe("with email-password auth", () => {
     importAppBefore("with-email-password");
     it("throws on login with non existing user ", async function (this: Mocha.Context & AppContext & RealmContext) {
-      expect(this.app).instanceOf(Realm.App);
-      const credentials = Realm.Credentials.emailPassword("me", "secret");
+      expect(this.app).instanceOf(App);
+      const credentials = Credentials.emailPassword("me", "secret");
       let didFail = false;
       const user = await this.app.logIn(credentials).catch((err) => {
         expect(err.message).equals("invalid username/password");
-        expect(err.code).equals(50);
+        expect(err.code).equals(4349);
```

Comment:
I've already fixed this on https://github.com/realm/realm-js/pull/5595 .. perhaps I should get that merged soon 🤔